### PR TITLE
trailscraper 0.5.1 (new formula)

### PR DESCRIPTION
**Pull request by nitrocode**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I used the [python poet instructions](https://docs.brew.sh/Python-for-Formula-Authors) to setup this formula. 

```
pip install trailscraper homebrew-pypi-poet
poet trailscraper
```

Then I added the `depends_on` argument and a valid test. I installed it and ran the tests locally and it's working as expected.

### CODE REVIEW

File: Formula/trailscraper.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
+class Trailscraper < Formula
+  include Language::Python::Virtualenv
+
+  desc "Tool to get valuable information out of AWS CloudTrail"
+  homepage "https://github.com/flosell/trailscraper"
+  url "https://github.com/flosell/trailscraper/archive/0.5.1.tar.gz"
+  sha256 "6793353bef9cf17379d556e7811d225ee852c6c2e975deb7219d684be7b12c5b"
+
+  depends_on "python"
```
Comment (chenrui333) [inline Formula/trailscraper.rb:9]: Maybe you can use python 3.8?
```suggestion
  depends_on "python@3.8"
```

Comment (nitrocode) [reply Formula/trailscraper.rb]: Tested and updated

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
@nitrocode can you squash the commits into one with commit message as `trailscraper 0.5.1 (new formula)`? Thanks!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@nitrocode, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**